edx_repo_tools/modernize_openedx_yaml.py

```diff
@@ -0,0 +1,44 @@
+import click
+import sys
+from ruamel.yaml import YAML
+
+DEPRECATED_FIELDS = ['owner', 'supporting_teams', 'track_pulls', 'track-pulls']
+
+
+class YamlModernizer:
```

Comment:
A docstring here or at the module level might be useful for future reference.  pylint would normally complain about this, but it looks like we haven't enabled it for this repo yet...that's a project for another time.